models/articles.models.js

```diff
@@ -28,3 +28,18 @@ exports.fetchArticlesById = (id) => {
       }
     });
 };
+
+exports.fetchCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`, [id]
+    )
+    .then(({ rows }) => {
+    
+      if (rows.length === 0) {
```

Comment:
What would happen here if there was a valid article but it had no comments yet? We wouldn't want it to throw an error but simply pass back an empty array
To do this we would need to check if the article exists, and if it does, search for comments, and if the article didn't exist, throw a 404 (as you've done here)

Reply:
Can I get a nchelp, please? I sent a request there.